### PR TITLE
8-patch-api-reviews-review_id

### PR DESCRIPTION
**Pull request by nasben64**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -279,3 +279,56 @@ describe("7. POST /api/reviews/:review_id/comments", () => {
...
+  test("PATCH - 404 sends review does not exist error message when given a valid but non-existent id", () => {
+    const newComment = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/reviews/99")
+      .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("review not found!");
+      });
+  });
```
Comment (emilyb93) [inline __tests__/app.test.js:333]: What about a test for if inc_votes is not a number?

File: models/reviews.js [hunk 2]
```diff
@@ -34,3 +35,25 @@ exports.selectReviewById = (review_id) => {
...
+exports.updateReviewById = (review_id, newVote) => {
+  if (newVote === undefined) {
+    return Promise.reject({
+      status: 400,
+      message: "inc_votes can not be undefined",
+    });
+  }
+  return checkReviewExists(review_id)
+    .then(() => {
+      console.log("inside the model");
```
Comment (emilyb93) [inline models/reviews.js]: remove logs before merge

**Review by emilyb93 — CHANGES_REQUESTED** [review]
One more test please :)

File: __tests__/app.test.js (hunk 1 above)
Comment (nasben64) [reply __tests__/app.test.js:333]: added the test for non integer

File: models/reviews.js (hunk 2 above)
Comment (nasben64) [reply models/reviews.js]: removed the console.log

**Pull request merged**